client/src/modules/events/graphql/queries.ts

```diff
@@ -1,4 +1,55 @@
 import { gql } from '@apollo/client';
+export const DATA_PAGINATED_EVENTS_TOTAL_QUERY = gql`
+  query PaginatedEventsWithTotal($limit: Int, $offset: Int) {
+    paginatedEventsWithTotal(limit: $limit, offset: $offset) {
+      total
+      events {
+        id
+        name
+        description
+        start_at
+        invite_only
+        canceled
+        image_url
+        tags {
+          tag {
+            id
+            name
+          }
+        }
+        chapter {
+          id
+          name
+          category
+        }
+      }
+    }
+  }
+`;
+export const DATA_EVENTS_QUERY = gql`
+  query PaginatedEvents($limit: Int, $offset: Int) {
+    paginatedEvents(limit: $limit, offset: $offset) {
+      id
+      name
+      description
+      start_at
+      invite_only
+      canceled
+      image_url
+      tags {
+        tag {
+          id
+          name
+        }
+      }
+      chapter {
+        id
+        name
+        category
+      }
+    }
+  }
+`;
```

Comment:
This does not appear to be used. Can we delete it?

Reply:
done